v1/urls.py

```diff
@@ -15,4 +19,27 @@
 urlpatterns = router.urls + [
     path('token/', TokenObtainPairView.as_view(), name='token_obtain_pair'),
     path('token/refresh/', TokenRefreshView.as_view(), name='token_refresh'),
+    path('hello/', hello_world, name='hello-world'),
 ]
+
+schema_view = get_schema_view(
+    openapi.Info(
+        title="Django Workshop API",
+        default_version='v1',
+        description="Django Workshop API created using Swagger for generating documentation",
+        terms_of_service="https://github.com/tilda-center/django-workshop/blob/main/LICENSE",
+        contact=openapi.Contact(email="contact@example.com"),
+        license=openapi.License(name="BSD 2"),
+    ),
+    public=True,
+    permission_classes=(permissions.AllowAny,),
+)
+
+
+# TODO: Modify this as necessary!
+urlpatterns = [
+    # path('admin/', admin.site.urls),
+    # path('api/', include('myapp.urls')),
+    path('swagger/', schema_view.with_ui('swagger', cache_timeout=0), name='schema-swagger-ui'),
+    path('redoc/', schema_view.with_ui('redoc', cache_timeout=0), name='schema-redoc'),
+]
```

Comment:
empty line at the end! please check editor settings, on save it should add a new empty line at the end.